### PR TITLE
Basic PDF and HTML versions working

### PR DESCRIPTION
**Pull request by arcaldwell49**
Basic PDF and HTML outlays now working. Once merged, will work on style guide for copyeditors/typesetters.

### CODE REVIEW

File: _extensions/cik/_extension.yml [hunk 1]
```diff
@@ -14,39 +14,84 @@ contributes:
...
       # Content to add in header that your format is using
       header-includes: |
         \usepackage{orcidlink}
         \definecolor{mypink}{RGB}{219, 48, 122}
       template-partials:
         # Add here the partials your format is using
+        - "partials/doc-class.tex"
+        - "partials/before-body.tex"
+        - "partials/citations.tex"
+        - "partials/graphics.tex"
```
[the remark this answers is not in the document]
Comment (smnnlt) [reply _extensions/cik/_extension.yml:47]: Nevermind, don't remove partials/graphics.tex as this changes the scaling of images (I don't understand why exactly)

File: _extensions/cik/cik.cls [hunk 2]
```diff
@@ -1327,7 +1327,7 @@ at https://doi.org/\@doi\ or via http://\@jname.royalsocietypublishing.org.\par}
 \vspace*{-5.5pc}
 \noindent\begin{minipage}[!t]{50mm}
 \hskip-1pc
-\vbox to 242mm{{\vskip68pt\includegraphics[width=55mm,scale=0.6]{cik_logo.pdf}\vskip10pt
+\vbox to 242mm{ { \vskip68pt\includegraphics[width=55mm,scale=1]{cik_logo.pdf}\vskip10pt
 %{\fontfamily{\sfdefault}\fontseries{m}\fontshape{n}\fontsize{13.5}{15}\selectfont\raggedright 
 %\textcolor{jobcolor}{storkjournals.org}}\vskip10pt
 \if@citearticle@provided
```
Comment (smnnlt) [inline _extensions/cik/cik.cls:1333]: Link directly to CC-BY in copyright?

File: template.qmd [hunk 3]
```diff
@@ -1,181 +1,228 @@
...
+        name: University of Awesome Research
+        url: https://example.com
+    email: fake@fake.edu
+  - name: John Doe
+    affiliations:
+      - ref: some-inst
+doi: "10.51224/XXXXXXXXX"
+journal-editor: "Sam Somebody"
+article_type: Research
+notetf: false
```
Comment (smnnlt) [inline template.qmd:21]: Add a comment, that this enables/disables the editor printnote

**Review by smnnlt — COMMENTED** [review]
Great work!

I've made some minor comments and suggestion, mainly regarding the removal of unneccessary intermediate files and partials. Next step after merging would be to update the README and license.

File: _extensions/cik/partials/before-body.tex [hunk 4]
```diff
@@ -3,18 +3,88 @@
...
+  {$by-author.name.literal$}
+$endfor$
+
+$for(by-affiliation)$
+\affil[$it.number$]{$it.name$}
+$endfor$
+
+$for(by-author)$
+$if(by-author.email)$
+  \corrauthor[*]{$by-author.email$}
```
[the remark this answers is not in the document]
Comment (arcaldwell49) [reply _extensions/cik/partials/before-body.tex:31]: True, it should go in the style guide

[the remark this answers is not in the document]
Comment (arcaldwell49) [reply _extensions/cik/partials/before-body.tex:31]: Also, the idea is only the corresponding author has their email added

**Review by smnnlt — APPROVED** [review]

**Pull request merged**